Prox/Prox/Map View/MapViewController.swift

```diff
@@ -131,22 +154,13 @@ class MapViewController: UIViewController {
     }
 
     @objc private func closeWithButton() {
-        resetMapToUserLocation(shouldAnimate: true)
-
-        // HACK: (no UX input) Delay the dismiss so the user can watch the animation and see what happens.
-        // TODO: we should change the delay depending on how far we animate.
-        // Note: I tried to add a listener for the end of the animation to dismiss but it complicated the code too much
-        // so here's this hack.
-        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + DispatchTimeInterval.milliseconds(500)) {
```

Comment:
Talking to @antlam, he agrees that the return-to-position effect isn't worth the lag the user feels after pressing the close button, so keeping rm'ed.